game/theater/iadsnetwork/iadsnetwork.py

```diff
@@ -120,6 +120,13 @@ def skynet_nodes(self, game: Game) -> list[SkynetNode]:
                 # Skip culled ground objects
                 continue
 
+            # HOTFIX! Skip non-static nodes with no alive units left
+            # Delete this as soon as PRs #2285, #2286 & #2287 are merged
+            unit_count = len(node.group.units)
+            is_static = node.group.units[0].is_static if unit_count > 0 else False
+            if node.group.alive_units == 0 and not is_static:
+                continue
+
```

Comment:
@RndName This should do the trick, thx for the heads-up though ;)

Reply:
top, thanks :)